### ISSUE TITLE
Where is the backend code?

### ISSUE THREAD

**Issue by Isaiah-Day** [CodeForBuffalo/buffalo_mutual_aid#5]
I'm not very good at Django, so i can't locate the python and HTML files that are run. I was looking at the code, but i don't know where the actual python files are. Can someone tell me? Thanks!

**Comment by Horrgs** [comment]
Then I recommend familiarizing yourself with Django, you can build a basic website to present data. The python files are included in the repository, you have to get an understanding of how Django works & runs in order to understand both those things. A brief ELI5 is that you run the manage.py file [here](https://github.com/CodeForBuffalo/buffalo_mutual_aid/blob/master/manage.py). 
The manage.py will look for the settings. My Django may be out of date, or done differently. In my case, I have a settings.py. In the case [here](https://github.com/CodeForBuffalo/buffalo_mutual_aid/tree/master/buffalomutualaid/settings), you have a settings directory filled with 4 python files, which comprise the entire settings. 
Next, the settings will look for the all registered URLs, [here](https://github.com/CodeForBuffalo/buffalo_mutual_aid/blob/master/buffalomutualaid/urls.py). The URLs point to the views.py for each URL. A view is the connection between the front-end and back-end. You can see [here](https://github.com/CodeForBuffalo/buffalo_mutual_aid/blob/master/search/views.py) where the website receives a search request, and does a GET request based off the search.

**Comment by Isaiah-Day** [comment]
Thanks!
